app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
Nice Work! Please also make it confirmable! So that user have to confirm before!

Reply:
@namangupta01 not sure what you mean about github configuration? can you help please?
I will make it confirmable too now.